src/napari_spatialdata/_view.py

```diff
@@ -32,14 +34,44 @@
 
 from napari_spatialdata._annotationwidgets import MainWindow
 from napari_spatialdata._model import DataModel
-from napari_spatialdata._scatterwidgets import AxisWidgets, MatplotlibWidget
-from napari_spatialdata._widgets import AListWidget, CBarWidget, ComponentWidget, RangeSliderWidget, SaveDialog
+from napari_spatialdata._scatterwidgets import AxisWidgets, PlotWidget
+from napari_spatialdata._widgets import (
+    AListWidget,
+    CBarWidget,
+    ComponentWidget,
+    RangeSliderWidget,
+)
 
 __all__ = ["QtAdataViewWidget", "QtAdataScatterWidget"]
 
 from napari_spatialdata.utils._utils import _get_init_table_list, block_signals
 
 
+class AnnotationDialog(QDialog):
```

Comment:
```suggestion
class ScatterAnnotationDialog(QDialog):
```
Perhaps this is better to avoid ambiguity regarding what annotation is meant as we now also have the annotation widget:)